.gitignore

```diff
@@ -1,5 +1,7 @@
 node_modules
 public/css/main.css
+public/css/content.css
```

Comment:
We don't generate content.css, so we should be save skipping it.

Reply:
So, how to make it not to generate?

Reply:
I mean, you can remove this line, because `_content.css ` is being generated as part of `main.css` and `homepage.css`, as we don't explicitly include `content.css ` into the page, we don't need this exception in `.gitignore`, so if I understand correctly you can just remove this line.

Reply:
Although it's a partial file somehow it's generating in the public/css/ directory. I don't know how to deal with it.

Reply:
That's weird, I can't reproduce it on my side, can you confirm that after deleting the generated `public/css/content.css ` it's keep getting generated on build?

Reply:
Sorry for inconsistency, but I think deleting one more time resolved this issue.